<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComentarU</class>
 <widget class="QMainWindow" name="ComentarU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(216, 229, 158);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnRegresar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>720</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>REGRESAR</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>891</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>PUBLICACION A COMENTAR</string>
    </property>
    <widget class="QLabel" name="lblUsuario">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>USUARIO: </string>
     </property>
    </widget>
    <widget class="QLabel" name="lblFecha">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>FECHA: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>TEXTO PUBLICACION: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUsuario">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plnText">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>711</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblImagen">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 20pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFecha">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>321</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>891</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Sitka&quot;;</string>
    </property>
    <property name="title">
     <string>COMENTARIO</string>
    </property>
    <widget class="QPlainTextEdit" name="txtComentario">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>511</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnComentar">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>180</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 900 10pt &quot;Segoe UI Black&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;
</string>
     </property>
     <property name="text">
      <string>COMENTAR</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="txtHora">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>115</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblCorreo">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>HORA:</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="txtFechaa">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>60</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;
border: 5px solid white;
background-color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="text">
      <string>FECHA:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
